src/ProjectTemplates/Web.ProjectTemplates/SignalR-CSharp.csproj.in

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+      <SignalRScripts Include="node_modules/@aspnet/signalr/dist/browser/signalr.js"/>
+  </ItemGroup>
+
+  <Target Name="NpmInstall"
```

Comment:
Uh, what? Why is this in the template?

Reply:
@BrennanConroy made a good point that using the CDN would hard-code us to a version. we'd prefer pull from NPM. do we want to make the user do this explicitly? seems like a barrier. this makes it automatic on build.

Reply:
We cannot put this in the project file. The user should run npm themselves.

Reply:
@davidfowl i talked this over with @anurse and @BrennanConroy and we're very much wanting the customer to have a successful experience doing two steps:

1. dotnet new signalr
2. dotnet run

When i look at the spa templates they're doing something similar by executing NPM in the .csproj file. Since we've got a precedent there for doing this, could we more closely follow that approach here? We really don't want to force customers to do this on their own.

Reply:
I feel pretty strongly about not putting this in the project file.

Reply:
okay. let's chat about a compromise. i'll set it up.

Reply:
😬

Reply:
:) I feel there's gotta be a way to make it easy for folks. even if we warn them on creation or have a comment somewhere. just want to make sure we're in agreement on *how* we make it easy.

Reply:
that doesn't entirely stink, but i still adhere to this being our goal:

dotnet new signalr
dotnet run
happy dev

I think we should strive to get to that point if possible with the template, in much the same way the spa templates achieved this.

Reply:
I do think we should be consistent here. If we don't want to run npm commands in our projects, we should remove it from the SPA templates too.